railties/lib/rails/command.rb

```diff
@@ -89,7 +89,7 @@ def root
       end
 
       def print_commands # :nodoc:
-        commands.each { |command| puts("  #{command}") }
+        commands.each { |command| puts("bin/rails  #{command}") }
```

Comment:
This method prints part of the output for the `rails help` command, but it comes after [this template](https://github.com/rails/rails/blob/6d6d4b9243144e2f2ea06d691b7145b7975edad2/railties/lib/rails/commands/help/USAGE) is rendered, so this change makes the format inconsistent:

```
$ bin/rails help
The most common rails commands are:
 generate     Generate new code (short-cut alias: "g")
 console      Start the Rails console (short-cut alias: "c")
 server       Start the Rails server (short-cut alias: "s")
 test         Run tests except system tests (short-cut alias: "t")
 test:system  Run system tests
 dbconsole    Start a console for the database specified in config/database.yml
              (short-cut alias: "db")

 new          Create a new Rails application. "rails new my_app" creates a
              new application called MyApp in "./my_app"


All commands can be run with -h (or --help) for more information.
In addition to those commands, there are:

bin/rails  about
bin/rails  action_mailbox:ingress:exim
bin/rails  action_mailbox:ingress:postfix
# etc.
```

I also think it's fine that `rails help` prints command names without `bin/rails` in front of them - we're fixing `rails test` because it's trying to give the user an executable command they can copy and run, and `rails -T` because we don't have much control over the format as it delegates to rake.

I really think we should back out the changes to the output of `rails test`, `rails -T`, and `rails help`, merge this PR, and then discuss what to do with these cases in a followup PR.

Thanks for your perseverance here!

Reply:
Yep, agreed. Let’s keep this to just the documentation changes.